bigbluebutton-html5/imports/api/captions/server/handlers/captionReply.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'underscore';
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+import { inReplyToHTML5Client } from '/imports/api/common/server/helpers';
+import addCaption from '../modifiers/addCaption';
+
+const CAPTION_CHUNK_LENGTH = 1000;
+
+export default function handleCaptionReply({ payload }) {
+  if (!inReplyToHTML5Client({ payload })) {
+    return;
+  }
+
+  const meetingId = payload.meeting_id;
+  const locale = payload.locale;
+  const captionHistory = payload.caption_history;
+
+  check(meetingId, String);
+  check(captionHistory, Object);
```

Comment:
Perhaps add check(locale, String)? I know we check locale once we call addCaption but still...